### ISSUE TITLE
Backtester error

### ISSUE THREAD

**Issue by brucetus** [DeviaVir/zenbot#566]
I'm trying to use the backtester.js script and I'm getting this error

```
[ 175/176 ] zenbot.sh sim  --trend_ema=20 --oversold_rsi=34 --oversold_rsi_periods=25 --neutral_rate=0.1 --period=2m  --min_periods=52
zenbot.sh sim  --trend_ema=20 --oversold_rsi=33 --oversold_rsi_periods=25 --neutral_rate=0.1 --period=2m  --min_periods=52
/bin/sh: 1: zenbot.sh: not found

[ 176/176 ] zenbot.sh sim  --trend_ema=20 --oversold_rsi=35 --oversold_rsi_periods=25 --neutral_rate=0.1 --period=2m  --min_periods=52
zenbot.sh sim  --trend_ema=20 --oversold_rsi=34 --oversold_rsi_periods=25 --neutral_rate=0.1 --period=2m  --min_periods=52
/bin/sh: 1: zenbot.sh: not found

zenbot.sh sim  --trend_ema=20 --oversold_rsi=35 --oversold_rsi_periods=25 --neutral_rate=0.1 --period=2m  --min_periods=52
/bin/sh: 1: zenbot.sh: not found


Backtesting complete, saving results...

Results successfully saved to backtesting_1506370721.csv!
```
It says it was successful, however it didn't actually run any simulations and the .csv file it outputted is empty.

It says it can't find zenbot.sh. I tried running the command in the zenbot folder like this
`./scripts/auto_backtester/backtester.js` but it didn't solve anything.

Can somebody help me fix this? I tried npm install also and it didn't help. Thanks.

**Comment by DeviaVir** [comment]
Can you pull the latest master and retry?

**Comment by brucetus** [comment]
Yes I got the same error

**Comment by cmroche** [comment]
This will be due to the changes I add to fix docker script execution. The call to zenbot.sh needs to be prefixed to ./zenbot.sh since we are no longer wrapping node.

I’ll send a push request to fix it tonight, but this is already done in the genetic backtester if you want to compare and fix yourself.

Sorry for the trouble, forgot about this script when testing.

**Comment by cmroche** [comment]
Fixed in #579
